paddle/c/scope.cc

```diff
@@ -0,0 +1,32 @@
+#include "scope.h"
```

Comment:
This PR is just an exploring PR for how to expose Paddle C++ core by C-API & Cython. I will give another PR by using pybind11.